sdk/cosmosdb/cosmos/test/public/functional/sproc.spec.ts

```diff
@@ -260,5 +260,12 @@ describe("NodeJS CRUD Tests", function (this: Suite) {
       .execute(undefined, [], requestOptions);
     assert.equal(result3, "Success!");
     assert.equal(headers3[Constants.HttpHeaders.ScriptLogResults], undefined);
+
+    let maxIntegratedrequestOptions = { maxIntegratedCacheStaleness: 2000 };
+    const { resource: result4, headers: headers4 } = await container.scripts
+      .storedProcedure(retrievedSproc.id)
+      .execute(undefined, [], maxIntegratedrequestOptions);
+    assert.equal(result4, "Success!");
+    assert.equal(headers4[Constants.HttpHeaders.ScriptLogResults], undefined);
```

Comment:
Is ScriptLogResult related to the Interated cache? Or covering a missing assertion before?

Reply:
I may remove the code. I asked the team about this. All other SDKs only test if the header is set correctly.

Reply:
Do we have test in Node which test if header is sent over wire?